typed-racket-test/unit-tests/typecheck-tests.rkt

```diff
@@ -4204,6 +4204,7 @@
                    [else 42]))
                (void))
              -Void]
+       [tc-e (map string-trim (ann '("a" "b") (Listof String))) (-lst -String)]
        )
```

Comment:
should we add a negative test (i.e. that it doesn't work when mandatory keywords are present?)

Reply:
I don't think we should add negative tests for incompleteness.

Reply:
isn't it possible to write a test that _should fail_ here because of a non-provided mandatory keyword argument? (sorry if I'm missing something)

Reply:
Ah, I understand now. Added.